content/api/overview/overview.md

```diff
@@ -6,4 +6,6 @@ order: 1
 ## Overview
 This section details Datadogs HTTP API. It makes it easy to get data in and out of Datadog.
 
-The Datadog API uses resource-oriented URLs, uses status codes to indicate the success or failure of requests and returns JSON from all requests. Let's dive in and see how it works.
\ No newline at end of file
+The Datadog API uses resource-oriented URLs, uses status codes to indicate the success or failure of requests and returns JSON from all requests. Let's dive in and see how it works.
+
+**Note**: Curl code examples displayed work only with `coreutils`, if you are working on a MacOsX device, install it with `brew install coreutils`
```

Comment:
cURL code examples are intended for use on Linux systems. To run the examples on a Mac, you will need to install `coreutils` using the [homebrew package manager](https://brew.sh/): `brew install coreutils`

Reply:
Maybe:
cURL code examples assume usage of BASH and GNU coreutils. On OSX you can install coreutils via the Homebrew package manager: brew install coreutils